src/components/Nft/NftCollectionRow.tsx

```diff
@@ -51,14 +52,14 @@ function NftCollectionRow({ collection, onCollectionPress }: Props) {
           color={"neutral.c70"}
           ml={5}
         >
-          {collection.nfts.length}
+          {collection.length}
```

Comment:
Bascially, don't we want to `?` everything? :smile:

Reply:
Done 👌